backend/de.metas.payment.esr/src/main/java/de/metas/payment/esr/api/impl/ESRImportDAO.java

```diff
@@ -367,4 +379,27 @@ public I_ESR_ImportLine fetchLineForESRLineText(@NonNull final I_ESR_Import esrI
 				.collect(ImmutableSet.toImmutableSet());
 		return notReconciledESRImportIds;
 	}
+	
+	@Override
+	public Set<PaymentId> findExistentPaymentIds(@NonNull final I_ESR_ImportLine esrLine)
```

Comment:
if the method will return max 1 id, i better to make it return Optional or `..OrNull`